index.rst

```diff
@@ -21,301 +21,433 @@ Some documentation on this system can be accessed on `Docushare Collection-11943
+The electronics cabinet must be turned off before opening.
+Before turning the power off, please be sure to power down the control computer (auxtel-ill-control.cp.lsst.org).
+Inside the cabinet there is a thermostat and fan that keeps the electronics cooled. The thermostat is set at 20C. 
+
+The cabinet includes the control computer (auxtel-ill-control.cp.lsst.org) and the windows machine for the monochromator, both of which are embedded SBCs. 
+There are two serial device servers for the electrometer and chiller and the LabJack with custom board. 
+There are 5V, 12V and 24V voltage supplies, all of which plug into an 8-outlet PDU.
+
+The electronics cabinet is powered by 220Vac from the dome. 
+Additionally, 6 ethernet cables are needed to run into the electronics cabinet for: electrometer serial device server, chiller serial device server, control computer (auxtel-ill-control.cp.lsst.org), monochromator windows machine, LabJack, PDU. 
 
-The cabinet includes the control computer (loonie) and the windown machine for the monochromator, both of which are embedded SBCs. There are two serial device servers for the electrometer and chiller and the LabJack with custom board. There are 5V, 12V and 24V voltage supplies, all of which plug into an 8-outlet PDU.
+.. figure:: /_static/elec_cabinet.jpg
+   :name: elec_cabinet
+   :target: ../_images/elec_cabinet.jpg
+   :alt: elec_cabinet
 
-The electronics cabinet is powered by 220Vac from the dome. Additionally, 5 ethernet cables are needed to run into the electronics cabinet for: electrometer serial device server, chiller serial device server, control computer (loonie), monochromator windows machine, LabJack, PDU. 
+   View of electronic cabinet when opened. 
 
 .. figure:: /_static/AuxTel_Ill_Functional_Layout_2.png
    :name: functional-diagram
    :target: ../_images/AuxTel_Ill_Functional_Layout_2.png
    :alt: Functional Diagram
 
-The PDU outlets are ordered as follows:
-
-+--------+--------------------+
-| Outlet | Name               |
-+--------+--------------------+
-| 1      | Chiller            |
-+--------+--------------------+
-| 2      | Electrometer       |
-+--------+--------------------+
-| 3      | White Light Source |
-+--------+--------------------+
-| 4      | Fiber Spectrograph |
-+--------+--------------------+
-| 5      | Monochromator      |
-+--------+--------------------+
-| 6      | 5V                 |
-+--------+--------------------+
-| 7      | 12V                |
-+--------+--------------------+
-| 8      | 24V                |
-+--------+--------------------+
+   Functional diagram of AuxTel Calibration Illumination system.
+
+.. table:: The PDU outlet numbering
+
+   +--------+--------------------+
+   | Outlet | Name               |
+   +--------+--------------------+
+   | 1      | 24 V               |
```

Comment:
Didn't you want to add the monochromator to this?

Reply:
Sure, added that.